### ISSUE TITLE
django ldap automatic login

### ISSUE THREAD

**Issue by youxit5** [almet/django-auth-ldap#7]
Hi Alexis,

I am developing a web app within my company for my company's employees with Django. Our company has a Active Directory server. Let's say I have ldap backend set up with django-auth-ldap on my Django web app. Every time users go to my web app, they would need to enter their AD user name and password. Is there a way to automatically sign in the users after they log into their window PC?

Thank you

**Comment by morocarlo** [comment]
hi, have you find a way to solve this "problem"?

**Comment by zedingli** [comment]
No, Django official guide doesn't have enough information, either. I have been trying to find a solution but have not had time to get into it. Please let me if you find a solution. I am wondering if it needs a windows server to do this.

**Comment by morocarlo** [comment]
we are tring to use https://github.com/kvspb/nginx-auth-ldap
ngnix try to contact the ldap server and with a "magic" procedure the server append in the request the username.
Then in django with a middleware you can read this header and autheticate the user.

When we will finish to develop this part I'll publish the code

**Comment by eqx-fschottz** [comment]
Hey @morocarlo could you finish this?
